src/shell/shell.zig

```diff
@@ -1307,8 +1328,8 @@ pub const LexError = struct {
     /// Allocated with lexer arena
     msg: []const u8,
 };
-pub const LEX_JS_OBJREF_PREFIX = "~__bun_";
-pub const LEX_JS_STRING_PREFIX = "~__bunstr_";
+pub const LEX_JS_OBJREF_PREFIX = &[_]u8{8} ++ "__bun_";
+pub const LEX_JS_STRING_PREFIX = &[_]u8{8} ++ "__bunstr_";
```

Comment:
☝️ (learning): @zackradisic What's the benefit of this bit

Reply:
We decided to use `\b` because it's hard to create a string with that so there's less of a chance of conflicts

We don't have to worry about `\b` clearing console output because we never print out the final script string. Though if were to create a function to return the final script string we would probably need to do a post-processing step to turn `\b__bun_0` -> `[Object object]` or something

Reply:
And no other character choice can be used (one perhaps without this gotcha?)